tough/src/lib.rs

```diff
@@ -874,10 +889,105 @@ fn load_targets<T: Transport>(
 
     // Now that everything seems okay, write the targets file to the datastore.
     datastore.create("targets.json", &targets)?;
+    if let Some(delegations) = &mut targets.signed.delegations {
+        load_delegations(
+            transport,
+            snapshot,
+            metadata_base_url,
+            max_targets_size,
+            delegations,
+            &datastore,
+        )?;
+    }
 
     Ok(targets)
 }
 
+//Follow the paths of delegations starting with the top level targets.json delegation
```

Comment:
Style nit throughout: comments markers should have a trailing space.

```suggestion
// Follow the paths of delegations starting with the top level targets.json delegation
```